src/aspire/basis/fb_2d.py

```diff
@@ -242,8 +242,8 @@ def _evaluate_t(self, v):
             must equal `self.sz`.
         :return: The evaluation of the coefficient array `v` in the dual basis
             of `basis`. This is an array of vectors whose last dimension equals
-             `self.count` and whose first dimensions correspond to
-             first dimensions of `v`.
+            `self.count` and whose first dimensions correspond to
```

Comment:
Just curious, did this also cause trouble? (or just for consistency?)

Reply:
This was just for consistency. Any amount of indentation will work, but a docstring linter would align all indented lines to the least indented. I've chosen to indent a tab (or 4 spaces).